src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py

```diff
@@ -5746,6 +5746,49 @@ def test_sql_managed_db_mgmt(self, mi, rg):
                  .format(resource_group_1, managed_instance_name_1, database_name),
                  expect_failure=True)
 
+    @ManagedInstancePreparer()
+    def test_sql_midb_ledger(self, mi, rg):
```

Comment:
After adding test, pls commit the test recording file too.

Reply:
Added test recordings.